### DISCUSSION TITLE
Zoom -> DAW

### DISCUSSION THREAD

**Discussion by gabe-santos** [ExistentialAudio/BlackHole#84] [General]
I'm trying to capture the audio from both my mic and my device but the multi-output I created doesn't show up as an input source on zoom

**Comment by devinroth** [comment]
Multi-Output is an output so it won’t show up as an input. 

I use a DAW such as Logic to mix the mic and BlackHole. There are free options like Audacity. Give it a try.

**Comment by devinroth** [comment]
I created a bad walkthough (plan on improving it but that's what I have for now). Hopefully it will help you out. http://existential.audio/blackhole/support/#micsystem
